tools/topology/sof/pipe-smart-amplifier-playback.m4

```diff
@@ -69,6 +69,15 @@ C_CONTROLBYTES(Smart_amp Model, PIPELINE_ID,
         ,
         MODEL_priv)
 
+
+C_CONTROLBYTES_READONLY(Smart_amp Model_Get_params, PIPELINE_ID,
+        CONTROLBYTES_OPS(bytes, 258 binds the mixer control to bytes get handlers, 258, 258),
+        CONTROLBYTES_EXTOPS_READONLY(258 binds the mixer control to bytes get handlers, 260),
```

Comment:
The comment doesnt match the number 260?

Reply:
Done